inst/include/common/information.hpp

```diff
@@ -174,10 +174,10 @@ class Information {
       FIMS_LOG << "Checking for available fleet index data objects."
                << std::endl;
       // set index data
-      if (f->observed_index_data_id != -999) {
-        uint32_t index_id = static_cast<uint32_t>(f->observed_index_data_id);
-        data_iterator it = this->data_objects.find(index_id);
-        FIMS_LOG << "Input fleet index id = " << index_id << "." << std::endl;
+      if (f->fleet_observed_index_data_id_m != -999) {
+        uint32_t index_id_m = static_cast<uint32_t>(f->fleet_observed_index_data_id_m);
```

Comment:
[quibble]: adding `info_` before `index_id_m` would be more consistent with the `fleet_` preface

Reply:
added info_ before index_id_m